docs/src/modules/components/MarkdownElement.js

```diff
@@ -464,14 +464,46 @@ const Root = styled('div')(
       boxShadow: `inset 0 -2px 0 var(--muidocs-palette-grey-200, ${lightTheme.palette.grey[200]})`,
     },
     '& details': {
+      width: '100%',
+      padding: theme.spacing(1),
       marginBottom: theme.spacing(1.5),
-      padding: theme.spacing(0.5, 0, 0.5, 1),
+      overflow: 'clip',
```

Comment:
I don't get how the content could overflow

```suggestion
```

Reply:
Good catch, yeah! Forgot to add that after I changed the layout 👍